Library/Homebrew/service.rb

```diff
@@ -45,6 +45,9 @@ def f
       ).returns(T.nilable(Array))
     }
     def run(command = nil, macos: nil, linux: nil)
+      # Save parameters for serialization
+      @run_params ||= command || { macos: macos, linux: linux }.compact
```

Comment:
```suggestion
      @run_params ||= command 
      @run_params ||= { macos: macos, linux: linux }.compact
```

feels a little bit more typical/idiomatic/easy to follow

Reply:
Addressed in https://github.com/Homebrew/brew/pull/15007/commits/801ee5e4744ba4f046e01d442ffd749af2d163a5